packages/nimble-components/src/text-area/tests/text-area.foundation.spec.ts

```diff
@@ -0,0 +1,675 @@
+/* eslint-disable @typescript-eslint/no-unnecessary-type-assertion */
+/* eslint-disable prefer-const */
+/* eslint-disable @typescript-eslint/no-inferrable-types */
+/* eslint-disable @typescript-eslint/dot-notation */
+/* eslint-disable @typescript-eslint/semi */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable @typescript-eslint/quotes */
```

Comment:
I mean we own this code now. That's the implication of forking the template and copying the tests. A separate file helps compare to upstream changes but we own it and presumably ran prettier, etc. on it. I don't think we should disable all these lint rules

Reply:
I think it would be good to document our policies about forking more clearly in nimble-components/CONTRIBUTING under "Extend Fast Foundation". This could include:
1. keep FAST-sourced tests separate from tests of our functionality (the .foundation.spec.ts naming pattern which seems to be new in this PR and #1244)
2. update FAST-sourced code to follow NI coding conventions
3. link back to the original source

We've been inconsistent about all of these (for example, [fixture.spec.ts](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/utilities/tests/tests/fixture.spec.ts#L1) didn't do 2 but did do 3). I could go either way on any of them, but it would be good to pick a direction and write it down.